[BUG] Flaky test failure - v1.2.5#mixedClusterTest - yaml rest test - indices.get_field_mapping

**Describe the bug**
Seen in https://github.com/opensearch-project/OpenSearch/pull/2437#issuecomment-1064790619

Gradle run result:
[Log 3252](https://ci.opensearch.org/logs/ci/workflow/OpenSearch_CI/PR_Checks/Gradle_Check/gradle_check_3252.log)
[Reports 3252](https://ci.opensearch.org/logs/ci/workflow/OpenSearch_CI/PR_Checks/Gradle_Check/gradle_check_3252_reports.zip)

**To Reproduce**
I can't reproduce it locally. 🙂
The below shows in the log of Gradle check.
```
REPRODUCE WITH: ./gradlew ':qa:mixed-cluster:v1.2.5#mixedClusterTest' --tests "org.opensearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=indices.get_field_mapping/20_missing_field/Return empty object if field doesn't exist, but index does}" -Dtests.seed=9FC2465A92C8B157 -Dtests.security.manager=true -Dtests.jvm.argline="-XX:TieredStopAtLevel=1 -XX:ReservedCodeCacheSize=64m" -Dtests.locale=uk-UA -Dtests.timezone=Australia/Hobart -Druntime.java=17

org.opensearch.backwards.MixedClusterClientYamlTestSuiteIT > test {p0=indices.get_field_mapping/20_missing_field/Return empty object if field doesn't exist, but index does} FAILED
    java.lang.AssertionError: Failure at [indices.get_field_mapping/20_missing_field:13]: expected [2xx] status code but api [indices.get_field_mapping] returned [404 Not Found] [{}]
        at __randomizedtesting.SeedInfo.seed([9FC2465A92C8B157:179679803C34DCAF]:0)
        at org.opensearch.test.rest.yaml.OpenSearchClientYamlSuiteTestCase.executeSection(OpenSearchClientYamlSuiteTestCase.java:442)
        at org.opensearch.test.rest.yaml.OpenSearchClientYamlSuiteTestCase.test(OpenSearchClientYamlSuiteTestCase.java:415)
```

**Expected behavior**
test passes.

**Plugins**
none.

**Screenshots**
none.

**Host/Environment (please complete the following information):**
 - OS: [e.g. iOS]
 - Version [e.g. 22]

**Additional context**
Occurred since Mar.10th, 2022.

The BWC test running with OpenSearch v1.3.0 can also fail.
gradle command: `./gradlew ':qa:mixed-cluster:v1.3.0#mixedClusterTest' --tests "org.opensearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=indices.get_field_mapping/20_missing_field/Return empty object if field doesn't exist, but index does}" -Dtests.seed=2E22A45097655A52`
see https://github.com/opensearch-project/OpenSearch/pull/2438#issuecomment-1065435468

--- a/rest-api-spec/src/main/resources/rest-api-spec/test/indices.get_field_mapping/20_missing_field.yml
+++ b/rest-api-spec/src/main/resources/rest-api-spec/test/indices.get_field_mapping/20_missing_field.yml
@@ -1,9 +1,5 @@
 ---
 "Return empty object if field doesn't exist, but index does":
-  - skip:
-      version: "all"
-      reason: "AwaitsFix https://github.com/opensearch-project/OpenSearch/issues/2440"
-
   - do:
         indices.create:
           index: test_index
@@ -18,7 +14,5 @@
       indices.get_field_mapping:
         index: test_index
         fields: not_existent
-        ignore: 404    # ignore 404 failures for now
-                       # see: https://github.com/opensearch-project/OpenSearch/issues/2440
 
   - match: { 'test_index.mappings': {}}

